docs-source/content/userguide/managing-domains/choosing-a-model/_index.md

```diff
@@ -1,5 +1,5 @@
 +++
-title = "Choose a model"
+title = "Choose a domain home source type"
```

Comment:
@tbarnes-us I used your title suggestion but it seems a bit clunky to me. If "Choose a Model" is no longer accurate, would "Choose a domain home type" or "Choose a domain home source" work?

Reply:
Let's go with 'Choose a domain home type'.

Reply:
Wait. Let's ask Ryan.

Reply:
@rjeberhard ^^^

Reply:
In the domain resource, we have a field called domainHomeSourceType.  This was the best name @mriccell and I could come up with.

Reply:
@rosemarymarano I think that's our final answer.  Please update all references to this section so that they use the new title.  For example please update the ci/cd doc reference accordingly.

Reply:
@tbarnes-us Will do, then I'll close this comment.

Reply:
@rosemarymarano ?

Reply:
Done.